runtime/src/bank_forks.rs

```diff
@@ -145,6 +151,8 @@ impl BankForks {
             descendants.entry(parent).or_default().insert(root_slot);
         }
 
+        let migration_status = Arc::new(Self::initialize_migration_status(&root_bank));
```

Comment:
This will initialize the `migration_status` on startup with either the snapshot or genesis block.
In the next PR when we process our startup blocks from `Blockstore`, i'll potentially enable alpenglow if we run into a valid Alpenglow block.

The idea is that once the validator is done with process blockstore the `migration_status` is correctly initialized. Currently we only initialize `migration_status` using the `BankForks` root, so we have incorrect behavior regarding the startup blocks.